build_tools/cmake/iree_cc_library.cmake

```diff
@@ -299,7 +299,9 @@ function(iree_cc_library)
     message(STATUS "  + alias ${_PACKAGE_NS}::${_RULE_NAME}")
   endif()
   add_library(${_PACKAGE_NS}::${_RULE_NAME} ALIAS ${_NAME})
-
+  # if(TARGET ${_OBJECTS_NAME})
+  #   add_library(${_PACKAGE_NS}::${_RULE_NAME}.objects ALIAS ${_OBJECTS_NAME})
+  # endif()
```

Comment:
Intentionally added commented out? Remove?

Reply:
Yeah, tried and it doesn't work. Removed.